src/ui/ServerAdministration/index.tsx

```diff
@@ -335,23 +348,35 @@ class ServerAdministrationContainer
           which won't receive updates.`,
         retry: {
           label: 'Downgrade Realm Studio',
-          onRetry: this.onDowngradeStudio,
+          onRetry: () => this.onDowngradeStudio(version),
         },
       },
     });
   }
 
-  protected downgradedStudioUrl() {
-    const baseUrl = 'https://studio-releases.realm.io/v1.19.2-ros2';
-    const platform = os.platform();
-    if (platform === 'darwin') {
-      return `${baseUrl}/download/mac-dmg`;
-    } else if (platform === 'linux') {
-      return `${baseUrl}/download/linux-appimage`;
-    } else if (platform === 'win32') {
-      return `${baseUrl}/download/win-setup`;
-    } else {
-      return baseUrl;
+  protected downgradedStudioUrl(version: string | undefined) {
+    if (!version) {
+      version = '0.0.0';
+    }
```

Comment:
@nirinchev Going over this again, I see that this could have been expressed as a default of the `version` argument:

```
protected downgradedStudioUrl(version: string = '0.0.0') {
```